README.md

```diff
@@ -72,6 +72,8 @@ You can right-click in the editor viewport to show an inline nodes list to selec
 
 - Arch Linux users may need to manually install libxcrypt before chaiNner's integrated Python will correctly start up.
 
+- Linux users need to have xclip or, for wayland users, wl-copy installed to be able to use the clipboard functionality.
```

Comment:
I'm wondering if this should maybe note that it's specific to the clipboard functionality that's performed by nodes, otherwise people might assume that regular copying and pasting doesn't work